policy/modules/system/systemd.te

```diff
@@ -1009,14 +1012,38 @@ systemd_read_efivarfs(systemd_sysctl_t)
 #
 # systemd_coredump domains
 #
+
+# dac_read_search - (at least) to read /proc/<pid>/exe symlinks
```

Comment:
Is this really needed? I cannot find any process with the directory permissions different to 0555.

Reply:
Actually, I was looking at the wrong audit record when writing the description... `sys_ptrace` is the one required for `/proc/<pid>/exe`; and `dac_read_search` is needed for acessing `/proc/<pid>/fd` of a non-root process. Here is the full AVC:
```
type=PROCTITLE msg=audit(18.02.2021 22:35:22.772:1247) : proctitle=/usr/lib/systemd/systemd-coredump 231283 1000 100756 6 1613684122 18446744073709551615 omos.redhat.com 
type=PATH msg=audit(18.02.2021 22:35:22.772:1247) : item=0 name=/proc/231283/fd inode=3496460 dev=00:17 mode=dir,500 ouid=omosnace ogid=omosnace rdev=00:00 obj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 nametype=NORMAL cap_fp=none cap_fi=none cap_fe=0 cap_fver=0 cap_frootid=0 
type=CWD msg=audit(18.02.2021 22:35:22.772:1247) : cwd=/ 
type=SYSCALL msg=audit(18.02.2021 22:35:22.772:1247) : arch=x86_64 syscall=openat success=yes exit=1 a0=0xffffff9c a1=0x7ffeaae5aaa0 a2=O_RDONLY|O_NONBLOCK|O_DIRECTORY|O_CLOEXEC a3=0x0 items=1 ppid=2 pid=231284 auid=unset uid=root gid=root euid=root suid=root fsuid=root egid=root sgid=root fsgid=root tty=(none) ses=unset comm=systemd-coredum exe=/usr/lib/systemd/systemd-coredump subj=system_u:system_r:systemd_coredump_t:s0 key=(null) 
type=AVC msg=audit(18.02.2021 22:35:22.772:1247) : avc:  denied  { dac_read_search } for  pid=231284 comm=systemd-coredum capability=dac_read_search  scontext=system_u:system_r:systemd_coredump_t:s0 tcontext=system_u:system_r:systemd_coredump_t:s0 tclass=capability permissive=1 
```

Reply:
That makes a difference. Will you change the comment?

Reply:
Yes, it's reworded now.